test/mlir/accelerators/nnpa/conversion/onnx-to-zhigh/lstm.mlir

```diff
@@ -71,6 +71,45 @@ func @test_onnx_to_zhigh_ccfd0_reverse(%X: tensor<7x2000x204xf32>, %W: tensor<1x
+// CHECK:           [[VAR_8_:%.+]]:4 = "onnx.SplitV11"([[VAR_7_]]) {axis = 2 : si64} : (tensor<2x200x800xf32>) -> (tensor<2x200x200xf32>, tensor<2x200x200xf32>, tensor<2x200x200xf32>, tensor<2x200x200xf32>)
+// CHECK:           [[VAR_9_:%.+]] = "zhigh.StickForLSTM"([[VAR_8_]]#2, [[VAR_8_]]#0, [[VAR_8_]]#3, [[VAR_8_]]#1) : (tensor<2x200x200xf32>, tensor<2x200x200xf32>, tensor<2x200x200xf32>, tensor<2x200x200xf32>) -> tensor<*xf32>
+// CHECK:           [[hn_output_:%.+]], [[VAR_cf_output_:%.+]] = "zhigh.LSTM"([[VAR_3_]], [[VAR_cst_]], [[VAR_cst_]], [[VAR_6_]], [[VAR_1_]], [[VAR_9_]], [[VAR_2_]]) {direction = "bidirectional", hidden_size = 200 : si64, prev_layer = "none", return_all_steps = -1 : si64} : (tensor<7x2000x204xf32, #zhigh.encoding<{dataLayout = "3DS"}>>, none, none, tensor<*xf32>, tensor<*xf32>, tensor<*xf32>, tensor<*xf32>) -> (tensor<*xf32>, tensor<*xf32>)
+// CHECK:           [[VAR_10_:%.+]] = "zhigh.Unstick"([[hn_output_]]) : (tensor<*xf32>) -> tensor<7x2x2000x200xf32>
+// CHECK-DAG:       [[VAR_11_:%.+]] = "onnx.Constant"() {value = dense<-1> : tensor<1xi64>} : () -> tensor<1xi64>
+// CHECK-DAG:       [[VAR_12_:%.+]] = "onnx.Constant"() {value = dense<0> : tensor<1xi64>} : () -> tensor<1xi64>
+// CHECK-DAG:       [[VAR_13_:%.+]] = "onnx.Constant"() {value = dense<1> : tensor<1xi64>} : () -> tensor<1xi64>
+// CHECK-DAG:       [[VAR_14_:%.+]] = "onnx.Constant"() {value = dense<2147483647> : tensor<1xi64>} : () -> tensor<1xi64>
+// CHECK-NOT: separator of consecutive DAGs
+// CHECK-DAG:       [[VAR_15:%.+]]:2 = "onnx.SplitV11"([[VAR_10_]]) {axis = 1 : si64} : (tensor<7x2x2000x200xf32>) -> (tensor<7x1x2000x200xf32>, tensor<7x1x2000x200xf32>)
+// CHECK-DAG:       [[VAR_16_:%.+]] = "onnx.Slice"([[VAR_15_]]#0, [[VAR_11_]], [[VAR_14_]], [[VAR_12_]], [[VAR_13_]]) : (tensor<7x1x2000x200xf32>, tensor<1xi64>, tensor<1xi64>, tensor<1xi64>, tensor<1xi64>) -> tensor<1x1x2000x200xf32>
+// CHECK-DAG:       [[VAR_17_:%.+]] = "onnx.Slice"([[VAR_15_]]#1, [[VAR_12_]], [[VAR_13_]], [[VAR_12_]], [[VAR_13_]]) : (tensor<7x1x2000x200xf32>, tensor<1xi64>, tensor<1xi64>, tensor<1xi64>, tensor<1xi64>) -> tensor<1x1x2000x200xf32>
+// CHECK-DAG:       [[VAR_18_:%.+]] = "onnx.Concat"([[VAR_16_]], [[VAR_17_]]) {axis = 1 : si64} : (tensor<1x1x2000x200xf32>, tensor<1x1x2000x200xf32>) -> tensor<1x2x2000x200xf32>
+// CHECK-DAG:       [[VAR_19_:%.+]] = "onnx.SqueezeV11"([[VAR_18_]]) {axes = [0]} : (tensor<1x2x2000x200xf32>) -> tensor<2x2000x200xf32>
+// CHECK-DAG:       [[VAR_20_:%.+]] = "zhigh.Unstick"([[VAR_cf_output_]]) : (tensor<*xf32>) -> tensor<*xf32>
+// CHECK-DAG:       [[VAR_21_:%.+]] = "onnx.SqueezeV11"([[VAR_20_]]) {axes = [0]} : (tensor<*xf32>) -> tensor<2x2000x200xf32>
+// CHECK:           return [[VAR_10_]], [[VAR_19_]], [[VAR_21_]] : tensor<7x2x2000x200xf32>, tensor<2x2000x200xf32>, tensor<2x2000x200xf32>
+// CHECK:         }
+}
+
+// -----
+
```

Comment:
Could you please another tests for bidirectional LSTM  with unknown dimensions? The same for GRU and the same for zhigh-to-zlow tests for LSTM/GRU. Thanks!

Reply:
Conversion tests for bidirectional LSTM/GRU with unknown dimensions added. Thank you.